handlers.go

```diff
@@ -202,12 +202,7 @@ func PostBulkUrl(c *gin.Context) {
 			validity = 30
 		}
 		responses[index] = PostUrlUtil(longUrl, alias, validity)
+		go saveUrltoAnalyticsDB(responses[index], alias)
```

Comment:
Analytics not working for random shorturl in bulk. In bulk shortening for custom url an entry is made in the analytics db but for random short url no entry is made.